README.md

```diff
@@ -64,9 +64,10 @@ options:
   -p str              Filename of a model parameter TOML file
   --safe-emails bool  Generate safe email addresses
   --safe-urls bool    Generate safe urls
-```
+  --fake-uuids bool   Generate clearly fake UUIDs for records
```

Comment:
Could be worth noting what this is for, and how this works? Is it to make such UUIDs easy to spot / track when debugging?